app/containers/MapContainer/utils.js

```diff
@@ -57,6 +57,105 @@ export const valueOfCircle = (radius, range, config) => {
   return scale(radius);
 };
 
+export const filterFeatureMaxZoom = (feature, zoom) => {
+  if (
+    feature.properties && feature.properties.marker_max_zoom
+  ) {
+    return zoom <= parseInt(feature.properties.marker_max_zoom, 10);
+  }
+  return false;
+};
+
+// append point countries onto map
+export const getPointLayer = ({ data, config, markerEvents }) => {
+  const layer = L.featureGroup(null);
+  const {
+    indicator, mapOptions, mapSubject, maxValueCountries, styleType, valueToStyle, zoom,
+  } = config;
+  const events = {
+    mouseover: (e) => markerEvents.mouseover ? markerEvents.mouseover(e, config) : null,
+    mouseout: (e) => markerEvents.mouseout ? markerEvents.mouseout(e, config) : null,
+    click: (e) => (markerEvents.click ? markerEvents.click(e, config) : null),
+  };
+
+  const jsonLayer = L.geoJSON(data, {
+    pointToLayer: (feature, latlng) => {
+      const filterFeature = filterFeatureMaxZoom(feature, zoom);
+      const styles = getAreaLayer(feature, indicator, mapOptions, mapSubject, maxValueCountries, styleType, valueToStyle);
+      return L.circleMarker(latlng, {
```

Comment:
let's try be something like

```
const svgIcon = L.divIcon({
  html: `
<svg
  xmlns="http://www.w3.org/2000/svg"
  viewBox="0 0 28 30"
  width="28"
  height="30"
>
  <path d="m24,14.18c0-5.52-4.48-10-10-10S4,8.66,4,14.18c0,4.37,2.8,8.07,6.71,9.43l3.29,4.2,3.29-4.2c3.9-1.36,6.71-5.07,6.71-9.43Z" fill="${iconRingColor}"/>
  <circle cx="14" cy="14.18" r="8.18" fill="${iconCircleColor}"/>
</svg>`,
  className: "",
  iconSize: [28, 30], // may have to adjust this
  iconAnchor: [12, 40], // will have to adjust this
});
```

Reply:
of course we'd have to determine the fill color (or color of the inner circle) similar to the area style (depending on type of map) - the outer "ring" color should default to white and turn black if feature is active